### PR TITLE
Remove Ubuntu 14.04

### PR DESCRIPTION
**Pull request by redhatrises**
It's entering Extended Security Maintenance (ESM) phase for security patching only

### CODE REVIEW

**Comment by redhatrises** [comment]
OCP failures have nothing to do with this PR.

File: docs/manual/developer_guide.adoc [hunk 1]
```diff
@@ -520,7 +520,7 @@ We have multiple benchmarks in our project:
 |===
 
 The *Linux OS* benchmark describes Linux Operating System in general.
-This benchmark is used by multiple ComplianceAsCode products, eg. `rhel7`, `fedora`, `ubuntu1404`, `sle11` etc.
```
Comment (carlosmmatos) [inline docs/manual/developer_guide.adoc:523]: should this become 1604?

File: example/README.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ PRODUCT_TO_CPE_MAPPING = {
 </pre>
 <pre>
 ...
-MULTI_PLATFORM_LIST = ["rhel", "fedora", "rhosp", "rhv", "debian", "ubuntu",
+MULTI_PLATFORM_LIST = ["rhel", "fedora", "rhosp", "rhv", "debian",
```
Comment (carlosmmatos) [inline example/README.md:121]: I assume this `ubuntu` means any version?

File: docs/manual/developer_guide.adoc (hunk 1 above)
Comment (redhatrises) [reply docs/manual/developer_guide.adoc:523]: fixed

File: example/README.md (hunk 2 above)
Comment (redhatrises) [reply example/README.md]: fixed

**Comment by carlosmmatos** [comment]
@redhatrises LGTM

**Pull request merged**